modules/builds-docker-credentials-private-registries.adoc

```diff
@@ -115,3 +115,11 @@ This example uses `pullSecret` in a Source build, but it is also applicable
 in Docker and Custom builds.
 ====
 endif::[]
+
+. You can also link the pull secret to the *builder* service account instead of specifying the
+`pullSecret` field. Pull secrets linked to the *builder* service account can be automatically added
+to any build in the namespace.
```

Comment:
this is more or less accurate but i'd reword it.

"can be automatically added" is too vague.

what you need to convey is the behavior which is:  if an explicit pullsecret is not specified on the build, the system will look at secrets linked to the build's SA and determine if any of them contain auth entries that reference the registry that is the target of the pull and, if it finds one, will use it.

Note also that there's no requirement the build is using the "builder" SA, that's just a default, so that needs to be made clear also.  If they are using some other SA, they need to link the secrets to that other SA.